src/conf/settings.xml

```diff
@@ -881,7 +881,7 @@ p, li { white-space: pre-wrap; }
             <minInt>300</minInt>
             <showDisplay>0</showDisplay>
             <stepInt>20</stepInt>
-            <valInt>400</valInt>
+            <valInt>495</valInt>
```

Comment:
I was checking the key, achcchually (if you will :grin:) the frequency of B is 494 (or at least close to). I'm fine with either :slightly_smiling_face:.